tasker/core/task_executor_main.py

```
@@ -153,7 +154,27 @@ def __init__(self, task_file, log_dir='logs', dry_run=True, log_level='INFO',
+        if alert_on_failure:
+            # Expand path and validate
+            alert_path = os.path.expanduser(alert_on_failure)
+            if not os.path.isabs(alert_path):
+                # Make relative paths relative to current directory
+                alert_path = os.path.abspath(alert_path)
+
+            if os.path.exists(alert_path):
+                self.alert_script = alert_path
+                # Make script executable if it isn't already
+                try:
+                    os.chmod(self.alert_script, 0o755)
```

Ruff (0.14.1)

`os.chmod` setting a permissive mask `0o755` on file or directory

(S103)